app/assets/stylesheets/common/base/onebox.scss

```diff
@@ -625,8 +625,8 @@ aside.onebox.stackexchange .onebox-body {
   }
 }
 
-// whitelistedgeneric twitter labels
-.onebox.whitelistedgeneric {
+// allowlistedgeneric twitter labels
+.onebox.allowlistedgeneric {
```

Comment:
I think we need to keep support for the old onebox classes. Otherwise all existing posts will lose their styling. The onebox changes will only take effect for new posts or following a rebake.